### ISSUE TITLE
local variable 'host_mm_mode' referenced before assignment

### ISSUE THREAD

**Issue by MTrab** [wxt9861/esxi_stats#52]
**Describe the bug**
When trying to integrate a vCenter environment, I get the described error in the HA logs.

**To Reproduce**
Integrate to vCenter 6.7

**Expected behavior**
A functional integration, as vCenter i mentioned as compatible

**Screenshots**

**ESXi environment (please complete the following information):**
 - Host/vCenter: vCenter 6.7.0.42000

**Additional context**
From the logs:

```
Logger: custom_components.esxi_stats
Source: custom_components/esxi_stats/__init__.py:166
Integration: ESXi Stats (documentation, issues)
First occurred: 18. april 2021 10.36.00 (1137 occurrences)
Last logged: 11.48.43

local variable 'host_mm_mode' referenced before assignment
```

**Comment by h4nsen** [comment]
Seeing this on vCenter 7.0 as well. Connection to an individual ESXi instance works as expected.

**Comment by wxt9861** [comment]
Can you try [0.6.3beta](https://github.com/wxt9861/esxi_stats/releases/tag/0.6.3beta) to see if it solves the issue?

**Comment by MTrab** [comment]
Looks like it's working on the beta.
Got a bunch of entities anyway :)

**Comment by wxt9861** [comment]
Fixed in #55
